### PR TITLE
[0x0st] v0.04

### PR DESCRIPTION
**Pull request by bw1**
\+ selfcheck

### CODE REVIEW

**Comment by vague666** [comment]
What's `selfcheckhelperscript`?
If it is an external dependency it might be better to use the symbol directly, after making sure the script is loaded, ie
Irssi::Script::selfcheck::selfcheckhelperscript($args), you can copy the loaded check from intercept.pl for example

**Comment by bw1** [comment]
thx for review 

selfcheckhelperscript is a part of a check system 

cycle

**Review by vague666 — APPROVED** [review]
Look good visually, I assume you've tested it :)

**Comment by bw1** [comment]
<https://travis-ci.com/github/bw1/check-irssi-scripts-selftest>

File: scripts/0x0st.pl [hunk 1]
```diff
@@ -8,17 +8,18 @@
...
     name	=> '0x0st',
     description	=> 'upload file to https://0x0.st/',
     license	=> 'ISC',
     url		=> 'https://scripts.irssi.org/',
-    changed	=> '2020-04-12',
+    changed	=> '2021-01-13',
     modules => 'POSIX HTTP::Request::Common LWP::UserAgent Storable File::Glob',
     commands=> '0x0st',
+    selfcheckcmd=> '/0x0st -c',
 );
```
Comment (ailin-nemui) [inline scripts/0x0st.pl:23]: remove /

File: scripts/0x0st.pl [hunk 2]
```diff
@@ -157,6 +167,29 @@ sub cmd {
...
+			$s = 'Error: arg count';
+		} elsif ( $res[1] != 200 ) {
+			$s = "Error: HTTP status code ($res[1])";
+		} elsif ( $res[0] !~ m/^http/ ) {
+			$s = "Error: result ($res[0])";
+		}
+	}
+	Irssi::print("0x0st: selfckeck $s", MSGLEVEL_CLIENTCRAP);
+	my $schs_version = $Irssi::Script::selfcheckhelperscript::VERSION;
+    Irssi::command("selfcheckhelperscript $s") if (defined $schs_version);
```
Comment (ailin-nemui) [inline scripts/0x0st.pl:190]: fix no TAB

**Comment by bw1** [comment]
`tab` and `/`

**Pull request merged**